lighthouse-core/gather/gather-runner.js

```diff
@@ -319,8 +329,10 @@ class GatherRunner {
     const artifacts = {};
 
     // Nest LighthouseRunWarnings, if any, so they will be collected into artifact.
-    gathererResults.LighthouseRunWarnings = [gathererResults.LighthouseRunWarnings];
+    const uniqueWarnings = Array.from(new Set(gathererResults.LighthouseRunWarnings));
```

Comment:
would it be worthwhile to change `LighthouseRunWarnings` to be a set instead?

Reply:
eh, it's not clear when we would do the flip back to array needed for JSON.stringify, so I'm not necessarily sold. since we were already manipulating it here and aggregating across multiple passes, seems reasonable to de-dupe but let audits keep track of it on their own